test/data/manifests/fedora_32-x86_64-vmdk-boot.json

```diff
@@ -1,6 +1,6 @@
 {
   "boot": {
-    "type": "vmware"
```

Comment:
Is this right?

Reply:
I don't know but in the `format-request-map` for the test case generation it's set to `qemu`.  Also for fedora-33 it's already set to `qemu`.  Not sure if they're meant to be different, or if f33 is the wrong one.

Reply:
Ah, I see. The commit that moved from `qemu` to `vmware` did not change the generator: https://github.com/osbuild/osbuild-composer/commit/9cce43d384bb499c7667ba471cde08dd7f1b6a14

So I guess the correct fix is to change the generator here (could also file an issue and do that in a follow-up).

We may also want to drop the boot testing from the image-tests for those image types we have proper integration tests for. The image tests really mostly verify that osbuild itself doesn't change behavior (might not even belong in composer, but that's another topic). What do you think @atodorov, should we drop the boot tests from the image tests in favor of the integration tests (where we have them)?

Reply:
> So I guess the correct fix is to change the generator here (could also file an issue and do that in a follow-up).

Changing and regenerating.

Reply:
I think that the reason is that we're using vcenter in the internal network but the integration tests for Fedora run in aws.